modules/websearch/lib/search_engine.py

```diff
@@ -1727,20 +1726,14 @@ def get_synonym_terms(term, kbr_name, match_type, use_memoise=False):
     return dterms.keys()
 
 
-def wash_output_format(ouput_format, verbose=False, req=None):
+def wash_output_format(ouput_format):
     """Wash output format FORMAT.  Currently only prevents input like
     'of=9' for backwards-compatible format that prints certain fields
     only.  (for this task, 'of=tm' is preferred)"""
     if str(ouput_format[0:3]).isdigit() and len(ouput_format) != 6:
         # asked to print MARC tags, but not enough digits,
         # so let's switch back to HTML brief default
         return 'hb'
-    elif format in CFG_WEBSEARCH_BLACKLISTED_FORMATS:
-        if verbose:
-            write_warning("Selected format is not available through perform_request_search", req=req)
-            # Returning an empty list seems dangerous because you wouldn't know
-            # right away that the list is not supposed to be empty.
-        return 'hb'
```

Comment:
this should remain.. it looks like it's coming from `legacy`, but we did not deployed it.